### PR TITLE
Updating nf-core-modules

### PR DESCRIPTION
**Pull request by asp8200**
#684 

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/nf-core/vcf_qc.nf [hunk 1]
```diff
@@ -12,7 +12,7 @@ workflow VCF_QC {
 
     ch_versions = Channel.empty()
 
-    BCFTOOLS_STATS(vcf)
+    BCFTOOLS_STATS(vcf, target_bed)
```
Comment (SusiJo) [inline subworkflows/nf-core/vcf_qc.nf:15]: To make this work you would need the target_bed files and their tbi indices. So what you can do it pass along an empty `target_bed` and then it should run through.
```suggestion
    BCFTOOLS_STATS(vcf, [])
```

**Comment by SusiJo** [comment]
Hey @asp8200,
There was already a [PR](https://github.com/nf-core/sarek/pull/671) to update the modules which was not merged in the end. There were some questions and concerns of @maxulysse and @FriederikeHanssen. Are these clarified now?

**Comment by asp8200** [comment]
> Hey @asp8200, There was already a [PR](https://github.com/nf-core/sarek/pull/671) to update the modules which was not merged in the end. There were some questions and concerns of @maxulysse and @FriederikeHanssen. Are these clarified now?

@SusiJo : I wasn't aware of your PR. @maxulysse asked me to update the modules so I went ahead with that.

**Comment by SusiJo** [comment]
No please go ahead :) My PR did not update the other modules, like yours.

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - [#679](https://github.com/nf-core/sarek/pull/679) - Back to `dev`
+- [#685](https://github.com/nf-core/sarek/pull/685) - Updating the
```
Comment (SusiJo) [inline CHANGELOG.md:17]: Is there a statement missing here?

Comment (asp8200) [reply CHANGELOG.md:17]: Thanks. Good catch. 
```suggestion
- [#685](https://github.com/nf-core/sarek/pull/685) - Updating the nf-core modules used by Sarek.
```

File: subworkflows/nf-core/alignment_to_fastq.nf [hunk 3]
```diff
@@ -15,6 +15,7 @@ workflow ALIGNMENT_TO_FASTQ {
     take:
     input // channel: [meta, alignment (BAM or CRAM), index (optional)]
     fasta // optional: reference file if CRAM format and reference not in header
+    fasta_fai
```
Comment (maxulysse) [inline subworkflows/nf-core/alignment_to_fastq.nf:18]: Actually, we don't need to pass either fasta nor fai since we have bam files at this step, right?

Comment (maxulysse) [reply subworkflows/nf-core/alignment_to_fastq.nf:18]: ok, never mind, I can see that we pass `[]` to this subworkflow

**Review by maxulysse — APPROVED** [review]
LGTM.
Let's try out this `stageAs: "?/*"`

**Pull request merged**